Formula/frotz.rb

```diff
@@ -13,6 +13,11 @@ class Frotz < Formula
     sha256 "887efd67a050038ffb15df8c713eaa40fcbfcc69d36f58373981dba3ccae422b" => :yosemite
   end
 
+  resource "testdata" do
+    url "https://github.com/anag004/FrotzTestFile/blob/master/test.z5?raw=true"
```

Comment:
How about using file from the source repo ? https://gitlab.com/DavidGriffith/frotz/tree/master/src/test 
I think that would be more maintainable in the long term.

Reply:
@javian I did consider that. However, those files are designed to debug implementations of Z-machine interpreters and not test an installation. Consequently, these files require user interaction to run which I could not emulate through the `.rb` file.